test/libsolidity/syntaxTests/using/library_functions_attached_in_single_directive_inside_library.sol

```diff
@@ -0,0 +1,19 @@
+library L {
+    function publicFunction(uint) public pure {}
+    function externalFunction(uint) external pure {}
+    function internalFunction(uint) internal pure {}
+    function privateFunction(uint) private pure {}
+
+    using {publicFunction, externalFunction, internalFunction, privateFunction} for uint;
+
+    function f() public pure {
+        uint x;
+        x.publicFunction();
+        x.externalFunction();
+        x.internalFunction();
+        x.privateFunction();
```

Comment:
I it would be more intuitive to have these always ordered by visibility. Definitions and in `using for` too. An in other tests.

```suggestion
        x.externalFunction();
        x.publicFunction();
        x.internalFunction();
        x.privateFunction();
```